doc/source/getting_started/docker/windows_container.rst

```diff
@@ -0,0 +1,165 @@
+.. _ref_windows_docker:
+
+Geometry Service as a Windows Docker container
+==============================================
+
+.. contents::
+
+.. _ref_running_windows_containers:
+
+Running Windows Docker containers
+---------------------------------
+
+To run the Windows Docker container for the Geometry service, ensure that you follow
+these steps when installing Docker:
```

Comment:
```suggestion
Runn the Windows Docker container
----------------------------------

To run the Windows Docker container for the Geometry service, ensure that you follow
these steps when installing Docker:
```
Paragraph is singular container. Changed heading to singular--and used the simple verb form.